File: In_Javascript/ex3.js
// Oson yo'l bilan yechganda
function oraliq_1(x, y){
    return (x >= 40 && x <= 60 && y >= 40 && y <= 60) || (x >= 70 && y <= 100 && y >= 70 && y <= 100);
}

// if bilan yechganda
function oraliq_2(x, y){
    if(x >= 40 && x <= 60 && y >= 40 && y <= 60){
        return true;
    }else if(x >= 70 && y <= 100 && y >= 70 && y <= 100){
        return true;
    }else{
        return false;
    }
}

console.log(oraliq_1(50, 89));
console.log(oraliq_2(50, 89));

File: In_Ruby/ex3.rb
# Logika orqaliy yechganda.
def sonlar_orasida(x, y)
  if y >= 40 && y <= 60 && x >= 40 && x <= 60 || x >= 70 && x <= 100 && y >= 70 && y <= 100 
    true
  else
    false
  end
end

# Oson yo'l bilan yechgan.
def sonlar_orasida_2(x, y)
  x.between?(40, 60) && y.between?(40, 60) || y.between?(70, 100) && x.between?(70, 100)
end

puts sonlar_orasida(44, 6)
puts sonlar_orasida_2(50, 99)